### PR TITLE
[Governance] proposal status

### PR DESCRIPTION
**Pull request by zihan-aptos**
* proposals table: add icon for status column
* proposal page header
    * add icon for status
    * hide "Time Remaining" when the proposal is closed
* proposal page 
    * hide vote buttons when the proposal is closed
Screen Shot 2022-08-09 at 12 17 43 PM
Screen Shot 2022-08-09 at 12 18 33 PM
Screen Shot 2022-08-09 at 12 18 44 PM
Screen Shot 2022-08-09 at 12 19 25 PM

### CODE REVIEW

File: src/pages/Governance/utils.tsx [hunk 1]
```diff
@@ -44,3 +48,41 @@ export function getProposalState(proposal: Proposal): ProposalState {
...
   }
 }
+
+export function renderStatusIcon(proposalState: ProposalState) {
+  switch (proposalState) {
+    case ProposalState.SUCCEEDED:
+      return (
+        <CheckCircleOutlinedIcon
+          fontSize="small"
+          // color={primaryColor}
```
Comment (msavir) [inline src/pages/Governance/utils.tsx:58]: can we remove those comments?

File: src/pages/Governance/Proposal/Header.tsx [hunk 2]
```diff
@@ -47,6 +50,10 @@ function TimeRemainingComponent({
   proposal: Proposal;
   isOnMobile: boolean;
 }) {
+  if (isVotingClosed(proposal)) {
```
Comment (msavir) [inline src/pages/Governance/Proposal/Header.tsx:53]: not sure if we dont want to show anything... maybe we can display the time it was closed?

Comment (zihan-aptos) [reply src/pages/Governance/Proposal/Header.tsx:53]: I thought about it but with the current design there isn't a good way to display the time it was closed so I just hide it for simplicity. Let me add a TODO here and revisit after consulting Josh. Btw do you know what is the accurate "close time"?

File: src/pages/Governance/Proposal/card/Index.tsx [hunk 3]
```diff
@@ -22,7 +23,9 @@ export function ProposalCard({proposal}: ProposalCardProps) {
           />
         }
       >
-        <CastVoteSection proposalId={proposal.proposal_id} />
+        {!isVotingClosed(proposal) && (
```
Comment (msavir) [inline src/pages/Governance/Proposal/card/Index.tsx:26]: do we also want to hide the vote option when the user has already voted?

**Review by msavir — APPROVED** [review]
lgtm

**Pull request merged**